easybuild/easyconfigs/i/I-TASSER/I-TASSER-4.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = "PackedBinary"
+
+name = "I-TASSER"
+version = '4.0'
+
+homepage = 'http://zhanglab.ccmb.med.umich.edu/I-TASSER/'
+description = """I-TASSER is a set of pre-compiled binaries and scripts for protein structure and function
+modelling and comparison."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Can't download from the web site as the file is behind a rego wall.
+# The URL is, nevertheless, included here for reference.
+#source_urls = ['http://zhanglab.ccmb.med.umich.edu/cgi-bin/download_I-TASSER.cgi?ID=358515010=I-TASSER%(version)s.tar.bz2']
+sources = ['%(name)s%(version)s.tar.bz2']
+patches = ['%(name)s-%(version)s-local-mods.patch']
```

Comment:
This line has been removed. I seem to have not included it in my branch, and from its name I surmise that it contained changes only relevant to our site.